include/eosio/vm/allocator.hpp

```diff
@@ -379,8 +379,8 @@ namespace eosio { namespace vm {
       template <typename T>
       void alloc(size_t size = 1 /*in pages*/) {
          if (size == 0) return;
-         EOS_VM_ASSERT(page != -1, wasm_bad_alloc, "require memory to allocate");
```

Comment:
page is signed, right?

Reply:
Yes. That's why it changed to "page >= 0"